documentation/assemblies/assembly-deploying-auth-services.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// assembly-installing-manual-steps.adoc
+
+[id='deploying-auth-services-{context}']
+= Deploying authentication services
+
+{ProductName} requires at least one authentication service to be deployed. 
+ifndef::NoneAuthService[]
+The authentication service can be `standard` ({AuthProductName}) or `external` (not managed by {ProductName}).
```

Comment:
Is there a way to not say 'authentication service' again?  "It can be" or "It can be set to"